backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/configuration/CassandraConfiguration.java

```diff
@@ -205,7 +238,9 @@ public CassandraConfiguration build() {
                 fetchNextPageInAdvanceRow.orElse(DEFAULT_FETCH_NEXT_PAGE_ADVANCE_IN_ROW),
                 blobPartSize.orElse(DEFAULT_BLOB_PART_SIZE),
                 attachmentV2MigrationReadTimeout.orElse(DEFAULT_ATTACHMENT_V2_MIGRATION_READ_TIMEOUT),
-                messageAttachmentIdsReadTimeout.orElse(DEFAULT_MESSAGE_ATTACHMENT_ID_MIGRATION_READ_TIMEOUT));
+                messageAttachmentIdsReadTimeout.orElse(DEFAULT_MESSAGE_ATTACHMENT_ID_MIGRATION_READ_TIMEOUT),
+                consistencyLevelRegular.orElse(DEFAULT_CONSISTENCY_LEVEL_REGULAR),
```

Comment:
we have to warn if DEFAULT_CONSISTENCY_LEVEL_QUORUM and DEFAULT_CONSISTENCY_LEVEL_LIGHTWEIGHT_TRANSACTION is a weird combination

Reply:
define weird

Reply:
LOCAL_QUORUM + SERIAL or LOCAL_SERIAL + QUORUM or EACH_QUORUM + LOCAL_SERIAL I guess